internal/cli/get.go

```diff
@@ -364,7 +364,7 @@ func printTemplateVersionsTable(w *tabwriter.Writer, tvs ...api.TemplateVersion)
 }
 
 func printRepositoriesTable(w *tabwriter.Writer, repos ...api.Repository) {
-	fmt.Fprintln(w, "NAME\tREPOSITORY URL\tACCESSIBLE")
+	fmt.Fprintln(w, "NAME\tREPOSITORY URL\tTYPE\tACCESSIBLE")
```

Comment:
According to @fzdarsky  comment
```suggestion
	fmt.Fprintln(w, "NAME\tTYPE\tREPOSITORY URL\tACCESSIBLE")
```

Reply:
👍